about.html

```diff
@@ -216,31 +220,28 @@ <h3 class="featurette-heading">The Birth of FCCG</h3>
         -->
 
 
-          <h3 class="featurette-heading">Now and in the future</h3>
-          <p>blurb overview about things we're trying to do and why</p>
-          <p>Stuff we're doing currently: GWC, JFK, bootcamps, etc</p>
-          <img class="featurette-image img-responsive center-block" src="http://placehold.it/1300x300" alt="timeline graph of our history. make interactive in phase 2? (so they can get info on past events)">
-          <p>Stuff we're doing in the future
-
-          <br>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Officiis laboriosam sequi, sapiente iure molestias ea reprehenderit vero aliquam laborum dolor voluptate blanditiis porro, facere, inventore distinctio! Obcaecati, et. Est, aliquam?</p>
+          <h3 class="featurette-heading">Now and in the Future</h3>
+          <p>We want to provide free computer science education to every one.
+          Therefore, we've been trying to start mini clubs in schools to get
+          students excited about coding. Also, bootcamps are held to introduce
+          and get people hooked onto computer programming. Afterward, if individuals
+          are interested in learning more, we have meetups to entertain any
+          questions and welcome new community members.
+          <br />Currently, we are facilitating Girls Who Code, a weekly class
+          teaching girls how to code. A club exists at John F. Kennedy
+          high school, where students are able to learn about computer programming
+          during lunch. Development for a bootcamp in February is in progress, and
+          the theme will be game development. Ultimately, our goal is to build
+          a community of not only people who know how to develop software, but
+          also designers, artists, or musicians. We want to have a space where
+          people with different backgrounds are able to convene and talk about
+          code or commit to an activity together.</p>
```

Comment:
I'm liking this worded out list of stuffs of what we're doing/will be doing.

Maybe there should be an actual `<li>` list below for TLDR and/or to list other things that were not discussed?